apps/web/components/ui/WasabiRegionSelect.tsx

```diff
@@ -0,0 +1,43 @@
+import { Select } from "@chakra-ui/react";
+import React from "react";
+
+const regions = [
+    { name: "US East 1 (N. Virginia)", code: "us-east-1" },
+	{ name: "US East 2 (N. Virginia)", code: "us-east-2" },
+    { name: "US Central 1 (Texas)", code: "us-central-1" },
+	{ name: "US West 1 (Oregon)", code: "us-west-1" },
+    { name: "CA Central 1 (Toronto)", code: "ca-central-1" },
+    { name: "EU Central 1 (Amsterdam)", code: "eu-central-1" },
+    { name: "EU Central 2 (Frankfurt)", code: "eu-central-2" },
+    { name: "EU West 1 (London)", code: "eu-west-1" },
+    { name: "EU West 2 (Paris)", code: "eu-west-2" },
+    { name: "AP Northeast 1 (Tokyo)", code: "ap-northeast-1" },
+    { name: "AP Northeast 2 (Osaka)", code: "ap-Northeast-2" },
+    { name: "AP Southeast 1 (Singapore)", code: "ap-southeast-1" },
+    { name: "AP Southeast 2 (Sydney)", code: "ap-southeast-2" },
+];
+
+type Props = {
+	value: string;
+	onChange: (e: React.ChangeEvent<HTMLSelectElement>) => void;
+};
+
+const WasabiRegionSelect: React.FC<Props> = ({ value, onChange }) => {
```

Comment:
@9brian can you confirm if this is working? you said this was buggy in one of our previous standups, so wanted to verify if this is still the case

Reply:
It's still buggy, for whatever reason it sends an AWS error. Even if the correct region was selected.

Reply:
is this something we want merged into the repo now or should we wait until it is fixed?

FYI you and Nathan can continue contributing even after CodeDay ends. I am sure the owner would appreciate it.

Reply:
+1 to what @sabrinaspage said, I would love to work together even after summer. On a side note, what's the issue? I might be able to help with it.

Reply:
@faisalsayed10 We wanted to implement a similar dropdown menu that is used in the AWS drive page, to the Wasabi drive page. It's giving an error at the moment stating that the endpoint was not valid

Reply:
could you please check with the wasabi docs about how their endpoints look like?